src/windows-emulator/windows_emulator.hpp

```diff
@@ -161,11 +164,18 @@ class windows_emulator
         return this->use_relative_time_;
     }
 
+    emulator_callbacks& callbacks()
```

Comment:
why add a getter again?

Reply:
i removed them as they served no purpose, you can just access the raw object instead